hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ReplicationManager.java

```diff
@@ -493,6 +498,15 @@ public boolean isContainerReplicatingOrDeleting(ContainerID containerID) {
         .isContainerReplicatingOrDeleting(containerID);
   }
 
+  private ECContainerReplicaCount getECContainerReplicaCount(
+      ContainerInfo containerInfo) throws ContainerNotFoundException {
+    Set<ContainerReplica> replicas = containerManager.getContainerReplicas(
+        containerInfo.containerID());
+    List<ContainerReplicaOp> pendingOps =
+        containerReplicaPendingOps.getPendingOps(containerInfo.containerID());
+    return new ECContainerReplicaCount(containerInfo, replicas, pendingOps, 0);
```

Comment:
The zero at the end of the parameters is definitely not correct, but I don't know yet what is the correct value to put here.

Could you add a TODO - define maintenance redundancy for EC (HDDS-6975) here?

We will need to fix this in a couple of places I think.